Header header
string cmd
---
bool result
